apertium-tur-uzb.tur-uzb.dix

```diff
@@ -2470,13 +2481,19 @@
 <e><p><l>ölçül<s n="v"/><s n="iv"/></l><r>oʻlchan<s n="v"/><s n="iv"/></r></p></e>
 <e><p><l>örüş<s n="v"/><s n="iv"/></l><r>toʻqish<s n="v"/><s n="iv"/></r></p></e>
```

Comment:
"örüş" never heard of it. what to'qish means?

Reply:
To'qish means "to knit". 
"örüş" seems like not a Turkish word. 
Doesn't it somehow mean "otlak"?

Reply:
then it is ör->to'qish